content/en/_index.md

```diff
@@ -730,6 +661,80 @@ $dlp = new DlpServiceClient();
 {{< /tab >}}
 {{< /tabpane >}}
 
+#### Authentication {#authentication}
+
+Samples should always authenticate using [Application Default Credentials][ADC].
+Most clients do this automatically, and no special steps are required.
+
+If, for whatever reason, the code snippet requires specific credentials,
+then, apart from showing how to obtain such credentials, it should show
+hot to intialized a client with other than ADC.
+
+[ADC]:https://cloud.google.com/docs/authentication/production#automatically
+
+{{< tabpane langEqualsHeader=true >}}
+{{< tab header="Java" >}}
+// Most clients use ADC by default. However, if your application needs to showcase a specific
+// credential source, show users how to do that explicitly.
+GoogleCredentials credentials = GoogleCredentials.fromStream(new FileInputStream("/path/to/credentials.json"));
```

Comment:
Still using service account key here. Can we remove this?

Reply:
Yes, I left a note on the PR comments. I only modified C#'s code for illustration purposes so we can agree on the approach. Once we have agreed, I'll modify the rest. That's why this is marked as a draft PR.